packages/next/server/config-shared.ts

```diff
@@ -1,13 +1,15 @@
 import os from 'os'
 import { Header, Redirect, Rewrite } from '../lib/load-custom-routes'
-import { ImageConfig, imageConfigDefault } from './image-config'
+import {
+  ImageConfig,
+  ImageConfigComplete,
+  imageConfigDefault,
+} from './image-config'
 
-type NoOptionals<T> = {
-  [P in keyof T]-?: T[P]
+export interface NextConfigComplete extends Required<NextConfig> {
+  images: ImageConfigComplete
 }
```

Comment:
```suggestion
export type NextConfigComplete = Required<NextConfig> & {
  images: ImageConfigComplete
}
```

We should probably use `export type` instead of `export interface` here since thats what it was exporting before.

Reply:
Thanks!